### ISSUE TITLE
self.d_config.is_caching_logits=True about  results_T

### ISSUE THREAD

**Issue by Chen-Hao-fc** [airaria/TextBrewer#27]
大佬好，有一点不能理解
看到代码里面对 is_caching_logits 的解释
`is_caching_logits (bool): if ``True``, caches the batches and the output logits of the teacher model in memory, so that those logits will only be calcuated once. It will speed up the distillation process. This feature is **only available** for :class:`~textbrewer.BasicDistiller` and :class:`~textbrewer.MultiTeacherDistiller`, and only when distillers' ``train()`` method is called with ``num_steps=None``. It is suitable for small and medium datasets.`

但是在distiller_basic.py中，
#261   batch, cached_logits = batch
#276   results_T = {'logits':[logits.to(self.t_config.device) for logits in cached_logits]}
result_T只是将对应的batch to devide,貌似没有经过self.model_T  **be calcuated once**。还是说在哪里统一的计算了，是我弄错了吗？请不吝赐教！

**Comment by Chen-Hao-fc** [comment]
sorry, 看到后面理解了，膜拜大佬！！！，我断章取义了。。。。
